File: program/WordCount/Proc/kotlin/ToWordList.kt
import java.io.File;

fun main() {
    File("news.txt").forEachLine {
        var line=it.lowercase()
        line=line.replace(Regex("[.,!'()-]"),"")
        val words=line.split(Regex(" +"));
        for (word in words) {
            println(word);
        }
    }
}

File: program/WordCount/Proc/c#/ToWordList.cs
using System;
using System.IO;
using System.Text.RegularExpressions;

class ToWordList {
	static void Main() {
		using (var sr=new StreamReader("news.txt")) {
			string line;
			while ((line=sr.ReadLine())!=null) {
				line=Regex.Replace(line.ToLower(),"[.,!'()-]","");
				foreach (var word in Regex.Split(line," +")) {
					Console.WriteLine(word);
				}
			}
		}
	}
}
